### PR TITLE
add ssdeep reputation script

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

**Comment by meirwah** [comment]
@erezh31 what is the status of this PR?

**Comment by erezh31** [comment]
will finish

**Comment by meirwah** [comment]
@erezh31 any update on this?

**Comment by erezh31** [comment]
@meirwah playbook test pass, other tests failed. you can merge

File: Tests/schemas/playbook.yml [hunk 1]
```diff
@@ -131,6 +131,8 @@ mapping:
             type: int
           separatecontext:
             type: bool
+          note:
```
Comment (meirwah) [inline Tests/schemas/playbook.yml]: what is this part of the PR related?
Please update from master @erezh31

**Comment by erezh31** [comment]
@meirwah the test passed but other tests failed, can you please merge?
thanks

**Pull request merged**